### ISSUE TITLE
Telescope-like picker instead of command-line window

### ISSUE THREAD

**Issue by swoh816** [vim-ctrlspace/vim-ctrlspace#313]
For Telescope being a popular Neovim picker for its user-friendliness, its combination with CtrlSpace will create great synergy.
(Also, CtrlSpace in LazyVim somehow displays a window one line smaller than the original CtrlSpace window.)

**Describe alternatives you've considered**
Not necessarily Telescope, but a pop-up floating window like Telescope, such as [dressing](https://github.com/stevearc/dressing.nvim) or [telescope-ui-select](https://github.com/nvim-telescope/telescope-ui-select.nvim) could also be beneficial.

**Additional context**
Thank you so much for the awesome plugin, I really can't use other project management plugin other than CtrlSpace, it's only the UI part that I wish to have more improved.

**Version(s) (please complete the following information):**
 - OS: Ubuntu 22.04 LTS
 - Version: LazyVim (nvim 0.9.4)

**Comment by Konfekt** [comment]
Yes, wholheartedly agree, I am also stuck with CtrlSpace but the binary picker is by now dispensable, since Vim seems to have it built-in. Maybe  @girishji 's https://github.com/girishji/scope.vim/ gives an idea. Maybe https://github.com/yehuohan/popc serves you well in the meanwhile, I didn't come around to trying it out, yet

**Comment by swoh816** [comment]
@Konfekt Thanks for pointing me to the right plugins. I'm not very used to configuring Neovim (specifically Lazyvim), so I couldn't delve much into Scope.vim myself. However, as you said, Popc does an amazing job, and now I can use CtrlSpace in a floating window. Thanks so much!

One problem that's newly introduced by using Popc is that now I see the following error when I launch LazyVim:
```
Error detected while processing function ctrlspace#keys#Keypressed[14]..ctrlspace#keys#workspace#LoadOrSave[2]..<SNR>23_loadWorkspace[32]..ctrlspace#workspaces#LoadWorkspace[62]..<SNR>70_execWorkspaceCommands[26]..TabClosed Autocommands for "*"..function <SNR>31_tabCallback:
line   22:
E516: No buffers were deleted: noautocmd bdelete! 1
E516: No buffers were deleted: noautocmd bdelete! 4
```
This error occurs only when I turn on Popc. The error disappears if I turn off (mute) Popc. Somehow, the functions in CtrlSpace seem to have been affected by Popc.

I configured both CtrlSpace and Popc using Lazy.nvim, which uses lua script.

**Comment by swoh816** [comment]
In addition to the above error, Popc restores tabline, so that tabline appear at the top even though I set `vim.opt.showtabline = 0`.

**Comment by Konfekt** [comment]
> Error detected while processing function ctrlspace#keys#Keypressed[14]..ctrlspace#keys#workspace#LoadOrSave[2]..<SNR>23_loadWorkspace[32]..ctrlspace#workspaces#LoadWorkspace[62]..<SNR>70_execWorkspaceCommands[26]..TabClosed Autocommands for "*"..function <SNR>31_tabCallback:

This is best asked at https://github.com/yehuohan/popc/issues but it is likely that as a fork there are competing autocmds

Regarding 2., maybe [layers](https://github.com/yehuohan/popc/blob/master/autoload/popc/layer/exp.vim) are the answer
